### PR TITLE
[ti_rapid7] Update format_version to 3.0.0

### PR DESCRIPTION
**Pull request by kcreddy**
## What does this PR do?

- Update `format_version` to `3.0.0`
- Add `owner.type: elastic` to package manifest.
- Update indicator rules with predefined indices instead of all. i.e., with `"auditbeat-*", "endgame-*", "filebeat-*", "logs-*", "packetbeat-*", "winlogbeat-*"`. These are the indices as per Prebuilt Indicator Match Rules installed by Elastic

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/integrations/issues/7814

### CODE REVIEW

File: packages/ti_rapid7_threat_command/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.9.0"
+  changes:
+    - description: Update format version to 3.0.0. Add owner.type to package manifest.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/7910
```
Comment (andrewkroh) [inline packages/ti_rapid7_threat_command/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/7910
    - description: Change the ECS event.type value from 'info' to 'indicator' for the vulnerability data stream.
      type: bugfix
      link: https://github.com/elastic/integrations/pull/7910
```

**Review by andrewkroh — APPROVED** [review]

**Comment by mrodm** [comment]
Please wait to merge this PR @kcreddy , merging this branch would publish this package as GA with spec v3 but this spec is not GA yet.

@andrewkroh could you check with your team in case there are more PRs updating format_version ? To try to avoid more packages updating to format_version 3.0.0

Thanks!!


cc @jsoriano

**Comment by mrodm** [comment]
> Please wait to merge this PR @kcreddy , merging this branch would publish this package as GA with spec v3 but this spec is not GA yet.
> 
> @andrewkroh could you check with your team in case there are more PRs updating format_version ? To try to avoid publishing more packages updating to format_version 3.0.0
> 
> We are checking how this validation is performed in elastic-package
> 
> Thanks!!
> 
> cc @jsoriano

@kcreddy you could proceed with this PR, just be aware that the spec v3 is not GA yet and there could be still changes in that version (e.g. more validation rules to be applied).

Just as a note, remember to add the capabilities if this package has some special requirement.

Sorry for the inconveniences

**Comment by kcreddy** [comment]
> @kcreddy you could proceed with this PR, just be aware that the spec v3 is not GA yet and there could be still changes in that version (e.g. more validation rules to be applied).

Great, thanks for the confirmation! 

> Just as a note, remember to add the capabilities if this package has some special requirement.

I think as outlined in the [meta issue for SEI packages](https://github.com/elastic/integrations/issues/7810#:~:text=3.0%C2%A0%237814-,No%20elastic.capabilities%20constraints%20should%20be%20added.%20This%20will%20ensure%20our%20packages%20are%20available%20across%20observability%20and%20security%20projects.,-The%20general%20command), is to not specifically add capabilities

File: packages/ti_rapid7_threat_command/changelog.yml (hunk 1 above)
Comment (kcreddy) [reply packages/ti_rapid7_threat_command/changelog.yml:6]: Updated as suggested

**Pull request merged**